src/Java.Interop.Tools.Generator/Utilities/Report.cs

```diff
@@ -7,6 +7,7 @@ namespace Java.Interop.Tools.Generator
 	public class Report
 	{
 		public static int? Verbosity { get; set; }
+		public static Action<string>? OutputDelegate { get; set; }
```

Comment:
For consistency elsewhere, this should be an `Action<TraceLevel, string>`?  See also ab3c2b2679d986ed3036c4cfb492f8c8b33657db.